products/arangodb

```diff
@@ -0,0 +1,33 @@
+---
+title: ArangoDB
+category: db
+permalink: /arangodb
+releasePolicyLink: https://arangodb.com/master-services-agreement-inc-may-2023-cs/#eolpolicy
+changelogTemplate: "https://docs.arangodb.com/stable/release-notes/version-__RELEASE_CYCLE__/"
+releaseLabel: "ArangoDB __RELEASE_CYCLE__"
+eolWarnThreshold: 121
+releases:
```

Comment:
Releases must be ordered from the latest to the oldest. Here this is the opposite.